lblod/notulen-prepublish-service
# Feature/typed agendapoints

type is now visually included in the agenda, would probably benefit from some styling in GN and GN-publication.

to test: 
 * make sure you've pulled dev branch of app-gelinkt-notuleren and ran the migrations.
 * use https://github.com/lblod/frontend-gelinkt-notuleren/pull/119
